keystrokes/src/components/Auth/auth.js

```diff
@@ -33,7 +34,9 @@ class Auth extends React.Component {
 
 	render() {
 		return (
```

Comment:
might be worth deconstructing user if you're using it multiple times.
`const { user } = this.state`

Reply:
![ihnmotp](https://user-images.githubusercontent.com/16790895/72587044-b5901000-38a8-11ea-9f85-25014317441d.jpg)

Reply:
Im giving this a rewrite - will be in the next update